doc/source/contributor/environments/ci-aio.rst

```diff
@@ -11,8 +11,29 @@ Prerequisites
 * a CentOS Stream 8 or Ubuntu Focal 20.04 host
 * access to the Test Pulp server on SMS lab
 
-Setup
-=====
+Automated Setup
+===============
+
+Access the host via SSH.
```

Comment:
Acquire the Ansible Vault password for this repository, and store a copy at
``~/vault-pw``.